include/tester/core/Application.hpp

```diff
@@ -23,15 +23,15 @@ class Application
     ~Application() = default;
 
     void init(int serverPort, int softphoneManagerPort, const std::string& softphoneManagerDomain,
-        int softphoneManagerLogLevel, const std::string& db_domain)
+        int softphoneManagerLogLevel, const std::string& databaseDomain)
```

Comment:
I'd like you to test if passing `databaseDomain` by value and using `std::move` on it (in `VTDatabase` and in `Database` as well) works like this does.

Reply:
Done